api/desecapi/views/domains.py

```diff
@@ -80,6 +81,18 @@ def domain(self):
     def get_queryset(self):
         qs = self.request.user.domains
 
+        if (  # other conditions? check policies?
```

Comment:
revisit

Reply:
Filter regardless of `auto_policy`, only allow reading domains for which policies allow any kind of permission; if no policy exists, all domains are visible